index.html

```diff
@@ -184,5 +184,38 @@ <h2 class="playfairFont margTop1">About</h2>
       </div>
     </section>
   </main>
+
+  <section class="artDeco2">
+    <h2 class="playfairFont">Get Started</h2>
+
+    <p class="loremStyle">Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna.</p>
+
+    <form action="https://formspree.io/f/xknavqkz" method="post">
```

Comment:
- [ ] Nice one adding the `post` method, but it should be spelled in uppercase as "POST" 😉. for consistency's sake.